tensordict/tensordict.py

```diff
@@ -615,18 +615,20 @@ def pop(
                 )
         return out
 
-    def apply_(self, fn: Callable) -> TensorDictBase:
+    def apply_(self, fn: Callable, inplace: bool = True) -> TensorDictBase:
```

Comment:
It's a PyTorch convention that methods suffixed with `_` are inplace, so I don't think it makes sense to have the `inplace` keyword argument here.

Reply:
Restored as before